.clang-tidy

```diff
@@ -43,6 +42,7 @@ Checks: >
     readability-avoid-const-params-in-decls,
     readability-const-return-type,
     readability-container-size-empty
+# temp. disabled: bugprone-exception-escape
```

Comment:
Is this still necessary? Or did you manage to address the errors with your changes in `KadabraBetweenness` and `DynBFS`?

Reply:
Apparently, there too many errors so they will be addressed in a second PR by me, so that we can merge this quickly.

Reply:
Alright makes sense.

Reply:
Yes, the changed files are handling `modernize-use-emplace`, the other errors are more technical (and more numerous). Since we need the update to fix CI, fixing the remaining tidy issues is imho out of scope from this PR.